tools/build/ubuntu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh 
+apt-get install --no-install-recommends --yes build-essential cmake doxygen \
+    git libboost-all-dev libcoin80c libcoin-dev libeigen3-dev libgtkglext1-dev \
+    libgts-bin libgts-dev libkdtree++-dev libkml-dev libmedc-dev \
```

Comment:
> libgts-bin libgts-dev

There are several redundant dependencies here which can be removed since you are specifying the -dev dependency, for example libgts-dev will pull in libgts-bin, libxmuu-dev will pull in libxmuu1, libxmu-dev will pull in libxmu6.